templates/zerver/digest_base.html

```diff
@@ -11,6 +11,17 @@
             <div class="portico-page" id="digest-page-title">
                 <h1> Zulip Digest </h1>
             </div>
+            <div>
+                <form action="{{ url('digest_page') }}" method="post"
+                  class="form-inline">
+                    {{ csrf_input }}
+                    <label for="message-weight">Message weight:</label>
+                    <input type="number" id="message-weight" name="message_weight" value="1" >
+                    <label for="sender-weight">Sender weight:</label>
+                    <input type="number" id="sender-weight" name="sender_weight"  value="1">
```

Comment:
Is there a reason you mix `-`s` and `_`s between the names/ids here?

Reply:
Oh sorry that should be changed.

Reply:
I think we can replace with ```-``` here as the variable is passed to django form where the field is written with ```_``` and we can use ```-``` there as python does not permit variables to have ```-```. I see we have such behaviour at few other places as well.